pages/Home.php

```diff
@@ -20,7 +20,7 @@
                     if($userlist[$file->getCreatorID()-1]['nickname'] != '') $creator = $userlist[$file->getCreatorID()-1]['nickname'];
                     elseif($userlist[$file->getCreatorID()-1]['fullname'] != '') $creator = $userlist[$file->getCreatorID()-1]['fullname'];
                     else $creator = $userlist[$file->getCreatorID()-1]['username'];
-                    echo '<div class="recentFile">'.$creator.' uploaded <a download href="/pages/get.php?f='.$file->getID().'">'.$file->getFilename().'</a> in category <a href="'.Helpers::getFullPathByCategoryID($file->getCategory(), $categories).'" target="_blank" rel="noopener">'.Helpers::getCategoryByID($file->getCategory(), $categories)->getValue().'</a></div>';
+                    echo '<div class="recentFile">'.$creator.' uploaded <a download href="/pages/get.php?f='.$file->getID().'">'.$file->getFilename().'</a> in category <a href="'.Helpers::getFullPathByCategoryID($file->getCategory(), $categories).'" target="_blank" rel="noopener">'.Helpers::getCategoryByID($file->getCategory(), $categories)->getLabel().'</a></div>';
```

Comment:
Why open in new tab?

Reply:
interesting that you notice it now xD but i dont want the user to just have a blank page, when i can open a new tab, request the file and instantly kill the tab again, since the request is fulfilled

Reply:
the link that opens in a new tab points to the category, not the file to download. I rather expected the download link to open in a new tab, not opening the category page the file has been uploaded to

Reply:
oh you mean the second link. actually dunno, i thuought it would be nicer to have it open in a second tab for "later" viewing and still can browse from the home page like normally